### ISSUE TITLE
Tests aren't working properly? + maintenance.

### ISSUE THREAD

**Issue by BMaxV** [Moguri/panda3d-gltf#119]
Hi,

I'd like to help improve this, since my animations continue to not work and I'd like to know why.

To start, I don't think running the tests works as intended?

I can run pytest, but running the `pip install -e .[test]` before it ends in this not found error so I'm not sure if the "passing" tests are cool or not.

```
 Traceback (most recent call last):
        File "/usr/lib/python3/dist-packages/setuptools/config.py", line 419, in _parse_attr
          return getattr(StaticModule(module_name), attr_name)
        File "/usr/lib/python3/dist-packages/setuptools/config.py", line 26, in __init__
          spec = importlib.util.find_spec(name)
        File "/usr/lib/python3.10/importlib/util.py", line 94, in find_spec
          parent = __import__(parent_name, fromlist=['__path__'])
        File "/home/myuser/Documents/Python/panda3d-gltf/gltf/__init__.py", line 1, in <module>
          import panda3d.core as p3d
      ModuleNotFoundError: No module named 'panda3d'
```

I noticed a lot of stuff going on in converter that's hard to understand so I will try and untangle things a bit.

Is there a reason there are lots of function definitions inside other functions?

**Comment by Moguri** [comment]
It seems you're running tests before dependencies are all installed?

**Comment by BMaxV** [comment]
panda is installed, I'm not sure why it says it can't be found.

**Comment by Moguri** [comment]
I am thinking about closing this issue. I do not know how to debug why the `panda3d` module cannot be found in your environment. Also, something like Discord might be a better place to discuss any refactoring topics instead of having an issue linger in the tracker forever.

**Comment by BMaxV** [comment]
The big problem is that the documentation lacks information on how to actually get from a blender animation to a working setup in panda. See the other issue I linked yesterday.

But I don't have time to work on this right now (maybe in 2 months) and I appreciate your work and your help, so I don't want to be annoying.

So I'm going to close the issue now and maybe reopen it when I have time and we talked about it on discord.
